### ISSUE TITLE
How to start the demo page?

### ISSUE THREAD

**Issue by wuct** [timberio/react-facet-filter#2]
I execute following commands

```
npm install
npm start
```

then run into this error

```
ERROR in Template execution failed: TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined

ERROR in   TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined



  - index.js:255
    [react-facet-filter]/[html-webpack-plugin]/index.js:255:16

  - util.js:16 tryCatcher
    [react-facet-filter]/[bluebird]/js/release/util.js:16:23

  - promise.js:504 Promise._settlePromiseFromHandler
    [react-facet-filter]/[bluebird]/js/release/promise.js:504:31

  - promise.js:561 Promise._settlePromise
    [react-facet-filter]/[bluebird]/js/release/promise.js:561:18

  - promise.js:598 Promise._settlePromiseCtx
    [react-facet-filter]/[bluebird]/js/release/promise.js:598:10

  - async.js:143 Async._drainQueue
    [react-facet-filter]/[bluebird]/js/release/async.js:143:12

  - async.js:148 Async._drainQueues
    [react-facet-filter]/[bluebird]/js/release/async.js:148:10

  - async.js:17 Immediate.Async.drainQueues
    [react-facet-filter]/[bluebird]/js/release/async.js:17:14
```

Is there any steps I miss?

**Comment by zsherman** [comment]
strange, started fresh and it works for me – what version of node/npm are you running?

**Comment by wuct** [comment]
My enviroment is

```
npm@3.10.3
node@6.3.1
```

**Comment by wuct** [comment]
OK. It works now. I believe it has fixed by #5.
